packages/twenty-front/src/modules/users/components/UserProviderEffect.tsx

```diff
@@ -69,6 +70,9 @@ export const UserProviderEffect = () => {
   const setCurrentWorkspaceMembersWithDeleted = useSetRecoilState(
     currentWorkspaceDeletedMembersState,
   );
+  const setCurrentUserAvailableWorkspaces = useSetRecoilState(
```

Comment:
it seems that we set this state in multiple places and that we already have the information in currentUser

==> using a recoil Selector looks what we are looking for